src/dotnet/Program.cs

```diff
@@ -121,22 +121,47 @@ private static int ProcessArgs(string[] args)
                 ["test"] = TestCommand.Run
             };
 
+            int exitCode = 100;
+            
+            string arguments = string.Empty;
+
+
             Func<string[], int> builtIn;
             if (builtIns.TryGetValue(command, out builtIn))
             {
-                return builtIn(appArgs.ToArray());
+                exitCode = builtIn(appArgs.ToArray());
+                
+                appArgs.ToList().ForEach(a => { arguments += a + " "; });
+
+            }
+            else
+            {
+                CommandResult result = Command.Create("dotnet-" + command, appArgs, FrameworkConstants.CommonFrameworks.NetStandardApp15)
```

Comment:
Is this a naming concern, then? Command.Create().Execute() returning a CommandResult seems pretty obvious. Is there a better name we should use here? Are you thinking Command.Create().Execute().GetResult()? 

Using explicit types is very cumbersome, so I'd rather keep member names obvious instead of bringing back this style.